CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,27 @@
+<p align="center">
+  <a href="https://github.com/tempestphp/tempest-framework" target="_blank">
+
+![](.github/tempest-logo.svg)
+  </a>
+</p>
+<p align="center">
+    <a href="LICENSE" target="_blank">
+        <img alt="Software License" src="https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square">
+    </a> 
+    <a href="https://coveralls.io/github/tempestphp/tempest-framework?branch=main" target="_blank">
+        <img alt="Coverage Status" src="https://coveralls.io/repos/github/tempestphp/tempest-framework/badge.svg?branch=main">
+    </a>
+</p>
+
+
+## Contributor Code of Conduct
```

Comment:
Let's not add a CoC for now. This requires a more thorough discussion amongst core members, and not a copy/paste from other repositories.